boltdesignsystem/bolt
# update build tools to no longer "magically" pull in related pkg dependencies

## Jira
http://vjira2:8080/browse/BDS-2111

## Summary
Removes "magically" pulling in related Bolt dependencies in our Webpack build + removes extra comments generated in prod mode. As a result (mostly from the dependency-related update), this PR shaves off around 200KB off our total JS bundle size!

![image](https://user-images.githubusercontent.com/1617209/80484522-d810db00-8925-11ea-9358-4e02bfdd7acb.png)


## Details
This "feature" was originally intended to help catch and wire up undeclared dependencies within Bolt components which given how things have been trending, should no longer be required.

## How to test
- Confirm tests passing
- Double-check with @remydenton on if we'd expect any unforeseen side-effects from this downstream
